sample-compose/gradle/libs.versions.toml

```diff
@@ -0,0 +1,140 @@
+[versions]
+androidCompileSdk = "34"
+androidMinSdk = "24"
+androidTargetSdk = "34"
+androidVersionCode = "1"
+androidVersionName = "1.0.0"
```

Comment:
```suggestion
androidVersionName = "3.31.0"
```

Reply:
Fixed in [fe32e37](https://github.com/nimblehq/android-templates/pull/596/commits/fe32e37ec2c45ed610f5e693fcd2fe23bd3c77a4) 🙈